lib/eex/lib/eex/engine.ex

```diff
@@ -96,6 +96,9 @@ defmodule EEx.Engine do
 
   All other markers are not implemented by this engine.
   """
+  @spec handle_expr(Macro.t, '' | '=', Macro.t) :: Macro.t
```

Comment:
ok. i will change that one to `char_list`,

So, should the callback definition be change  as well?
https://github.com/elixir-lang/elixir/blob/6151f2ab1af0189b9c8c526db196e2a65c609c64/lib/eex/lib/eex/engine.ex#L32